scripts/ecs-anywhere-install.sh

```diff
@@ -592,6 +602,73 @@ wait-agent-start() {
     fail
 }
 
+exec-setup() {
+    find-copy-certs-exec
+    download-ssm-binaries-exec
+}
+
+find-copy-certs-exec() {
+    # Reference for locations: https://golang.org/src/crypto/x509/root_linux.go
+    CERTS_FILES_LOCATIONS=("/etc/ssl/certs/ca-certificates.crt"
+      "/etc/pki/tls/certs/ca-bundle.crt"
+      "/etc/ssl/ca-bundle.pem"
+      "/etc/pki/tls/cacert.pem"
+      "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
+      "/etc/ssl/cert.pem"
+    )
+    CERTS_PATH="/var/lib/ecs/deps/execute-command/certs"
+
+    echo "Copying certs for exec feature"
+
+    # Determine certs file
+    certs=""
+    if [ -n "$CERTS_FILE" ] && [ -f "$CERTS_FILE" ]; then
+        certs=$CERTS_FILE
+    else
+        if [ -n "$CERTS_FILE" ]; then
+            echo "Provided certs file does not exist, looking for certs in known possible locations"
+        fi
+        for f in "${CERTS_FILES_LOCATIONS[@]}"
+        do
+            if  [ -f "$f" ]; then
+                certs=$f
+                break
+            fi
+        done
+    fi
+
+    # Copy certs to exec directory
+    if [ -z "$certs" ]; then
+        echo "Could not find certificates. Please rerun with --certs-file and provide a valid path"
+        fail
+    else
+        echo "Using $certs"
+        mkdir -p $CERTS_PATH && cd $CERTS_PATH
```

Comment:
not sure if `cd` would break other assumptions in the script now or in the future. Perhaps it's more convenient to not change the current dir and use absolute paths?